btf_types/src/lib.rs

```diff
@@ -0,0 +1,196 @@
+//! Provide types implementation for BF interpreter.
+use std::error::Error;
+use std::fmt;
+use std::fs;
+use std::path::Path;
+
+/// Provide enum for all possible BF language instructions.
+#[derive(Debug)]
+pub enum RawInstructions {
+    /// Reprsent `>` symbol. Increment the data pointer by one (to point to the next cell to the right).
```

Comment:
All of these have the same typo (copy/paste?) `Reprsent` should be `Represent` 😁